quickstarts/mlops/aporia/config.yml

```diff
@@ -0,0 +1,65 @@
+id: 69fbc2a1-5b71-4c5f-bf0f-26cb07b02c37
+# Name of the quickstart (required)
+name: mlops-aporia
+
+# Displayed in the UI
+title: Aporia
+
+# Long-form description of the quickstart (required)
+description: |
+  Aporia’s integration with New Relic provides you with full access to a customized ML monitoring dashboard in New Relic. The dashboard contains six different charts: Most Active Models, Most Active Model Versions, Model Inferences, Average Numeric Inferences, Numeric Inferences Heatmaps, and Categorical Inferences for a comprehensive inferences investigation for all your models in production.
+
+  This integration also supports monitoring for almost all model use cases, including: fraud detection, NLP, Recommendations, Sales Forecast, Churn Prediction, Lead Prioritization, and Tabular Data.
+
+  By leveraging New Relic Alerts and Applied Intelligence, you will be able to monitor and manage alerts for all your operational needs. Find, troubleshoot, and resolve problems faster and automatically detect anomalies and combine related alerts and incidents to enable root cause analysis for any operational issues that may arise, even beyond the model itself.
+
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Aporia’s integration with New Relic enables data scientists and ML engineers to receive a combination of custom and smart alerts integrated cross-platform, providing a broader picture of the ML infrastructure and an easier, quicker way to resolve issues.
+
+
+# Support level: New Relic | Verified | Community (required)
+level: New Relic
+
+# Authors of the quickstart (required)
+authors:
+  - Aporia
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - mlops
+  - ml-ops
+  - ai
+  - aporia
+  - machine-learning
+  - machine learning
+  - monitoring
+  - data monitoring
+  - integrity
+  - data integrity
+  - observability
+  - artificial intelligence
+  - artificial-intelligence
```

Comment:
```suggestion
```

Reply:
@rudouglas Can you explain why should we remove the keywords with a dash? If I remove it, Aporia won't be listed if someone searches for "artificial-intelligence" (with a dash) in the Instant Observability dashboard.
Same for the other keywords you suggested removing.

Reply:
@maorkh4 yes that's actually correct, though the guidance is to use spaces instead of `-`. I think this is ok for now, we'll need to make some changes on our end to make this clearer, thanks